### PR TITLE
build: Fix lerna package json

### PR DESCRIPTION
**Pull request by MichaelParadis**
## Changes
Made pre-release check for jobber/fonts easier to understand and more robust

### Added

- 

### Changed
 - build: Made pre-release check for jobber/fonts easier to understand and more robust
 - build: Undid change that would break CI builds

### Deprecated

- 

### Removed

- 

### Fixed

- 

### Security

- 

## Testing

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: packages/components/CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,53 @@ menu: Changelog
...
+# [3.0.0-alpha.0](https://github.com/GetJobber/atlantis/compare/@jobber/components@2.90.0...@jobber/components@3.0.0-alpha.0) (2023-01-24)
+
+
+### Features
+
+* **components:** Update drawer, modal and card headings ([#991](https://github.com/GetJobber/atlantis/issues/991)) ([1d9ade2](https://github.com/GetJobber/atlantis/commit/1d9ade2ec2043c0dcd7b9415409e094fcd7d27cb))
+* **design:** BREAKING CHANGE: Implement new typefaces in Atlantis [JOB-46209] ([#952](https://github.com/GetJobber/atlantis/issues/952)) ([15f604c](https://github.com/GetJobber/atlantis/commit/15f604cc51ee6881a5ed9c1be01081c76cf5ce6e))
+
+
+### BREAKING CHANGES
```
Comment (chris-at-jobber) [inline packages/components/CHANGELOG.md:20]: Is this a manual changelog edit? 👀 If so, TIL we could do that

Comment (MichaelParadis) [reply packages/components/CHANGELOG.md:20]: There is a command to generate it

Comment (chris-at-jobber) [reply packages/components/CHANGELOG.md:20]: ✨

**Review by taylorvnoj — APPROVED** [review]
LGTM

**Pull request merged**